scripts/Interface/ui/sans_isis/beam_centre.py

```diff
@@ -155,6 +155,18 @@ def set_run_button_to_normal(self):
         self.run_button.setText("Run")
         self.run_button.setEnabled(True)
 
+    def enable_update_hab(self, enabled):
+        if enabled:
+            self.update_hab_check_box.setEnabled(True)
+            self.update_hab_check_box.setChecked(True)
+        else:
+            self.update_hab_check_box.setChecked(False)
+            self.update_hab_check_box.setEnabled(False)
```

Comment:
If `enabled` is True or False can this method be condensed to just:
```python
def enable_update_hab(self, enabled):
    self.update_hab_check_box.setEnabled(enabled)
    self.update_hab_check_box.setChecked(enabled)
```

Reply:
Yep, this was left over from testing when I thought that the checkboxes could only be checked/unchecked when they were enabled. I'll change this